packages/block-editor/src/components/block-alignment-control/stories/block-alignment-toolbar.story.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { BlockAlignmentToolbar } from '../';
+
+/**
+ * The `BlockAlignmentToolbar` component is used to render block alignment options in the editor. The different alignment options it provides are `left`, `center`, `right`, `wide` and `full`.
+ */
+const meta = {
+	title: 'BlockEditor/BlockAlignmentToolbar',
+	component: BlockAlignmentToolbar,
+	parameters: {
+		docs: { canvas: { sourceState: 'shown' } },
+	},
+	argTypes: {
+		value: {
+			control: { type: null },
+			description: 'The current value of the alignment setting.',
+		},
+		onChange: {
+			action: 'onChange',
+			control: { type: null },
+			description:
+				"A callback function invoked when the toolbar's alignment value is changed via an interaction with any of the toolbar's buttons. Called with the new alignment value (ie:`left`, `center`, `right`, `wide`, and `full`) as the only argument.",
+		},
+		controls: {
+			control: { type: null },
+			description: 'An array of available alignment controls.',
+		},
+		isCollapsed: {
+			control: { type: null },
```

Comment:
```suggestion
			control: 'boolean',
```

Let's make this parameter controllable.


https://github.com/user-attachments/assets/3a04692c-d28b-4441-98be-16a09603d9d8